src/core/Akka.Cluster.Tests.MultiNode/SurviveNetworkInstabilitySpec.cs

```diff
@@ -43,7 +44,7 @@ public SurviveNetworkInstabilitySpecConfig()
             Seventh = Role("seventh");
             Eighth = Role("eighth");
 
-            CommonConfig = DebugConfig(false)
+            CommonConfig = DebugConfig(true)
```

Comment:
This is still a WIP for the moment - had some massive issues with Windows Defender going beserk and eating up 100% of CPU when I ran this locally; hoping the build server sheds some better light on it.